src/pg/configurator.rs

```diff
@@ -33,15 +53,16 @@ pub struct PgBuilder {
 impl PgBuilder {
     pub async fn new(config: &PgConfig, id_resolver: IdResolver) -> Result<Self> {
         let pool = Pool::new(config).await?;
-        let auto = config.run_autodiscovery;
+
         Ok(Self {
             pool,
             default_srid: config.default_srid,
-            auto_functions: config.auto_functions.clone().unwrap_or(Schemas::Bool(auto)),
-            auto_tables: config.auto_tables.clone().unwrap_or(Schemas::Bool(auto)),
+            calc_bounds: config.calc_bounds.unwrap_or(true),
```

Comment:
We can name it `disable_bounds` since this flag is intended to disable the default behavior (always expose bounds). We can also invert the default and use `enable_bounds`.